test/LeanCode.Tests.csproj

```diff
@@ -1,4 +1,26 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
What is the reason for this change?

Reply:
[Since .NET 6.0 preview1](https://github.com/dotnet/aspnetcore/commit/f7e1712db88db4c01fcc49e5995d9528dbf608df), `Microsoft.AspNetCore.Mvc.Testing` reads content roots from a JSON file first before consulting assembly attributes or solution directory. However, when that JSON is written, only projects referenced by the test projects are considered _without the test project itself_. So, because our `TEntryPoint` (Startup) class is in the test project itself, `WebApplicationFactory` [cannot find](https://github.com/dotnet/aspnetcore/blob/v6.0.0-preview.4.21253.5/src/Mvc/Mvc.Testing/src/WebApplicationFactory.cs#L196) its related `ContentRoot` and causes tests in `LeanCode.IntegrationTestHelpers.Tests` to fail. I work around that by ensuring it never tries to read that path from file, which requires me to set it via environment variable beforehand.